packages/server/lib/middleware/ratelimit.middleware.ts

```
@@ -62,7 +62,8 @@ export const rateLimiterMiddleware = (req: Request, res: Response, next: NextFun
             }
 
             logger.error('Failed to compute rate limit', { error: err });
-            res.status(500).send({ error: { code: 'server_error', message: 'Failed to compute rate limit' } });
+            // If we can't get the rate limit (ex: redis is unreachable), we should not block the request
```

we could use `report()` now in that case instead of swallowing the error

---

we'd be flooding the error reporting though. I am for now making the assumption that if Redis is unreachable we will get alerted somehow via another channel